### ISSUE TITLE
make official tarball portable.

### ISSUE THREAD

**Issue by zaazbb** [retext-project/retext#344]
i want a portable tarball,  so i can install it or run it directly. 
so, i sugest that:
1, move all date file to share dir,  same as the installed dir.
```
ReTextSourceRoot/share/appdata/me.mitya57.ReText.appdata.xml
ReTextSourceRoot/share/applications/me.mitya57.ReText.desktop
ReTextSourceRoot/share/icons/hicolor/scalable/apps/icons/retext.svg
ReTextSourceRoot/share/retext/icons/*
ReTextSourceRoot/share/retext/locale/*.qm
```
2, add the source root dir to datadirs.
add `datadirs.append(join(sys.path[0], 'share', 'retext'))` to the  `initializeDataDirs()` function.

now, the tarball should be portable.

and, i want keep my python root dir clean. 
so, the share data dir under windows,  should be move to the package dir, like this `PythonXX\Lib\site-packages\ReText\share\date-files`.

**Comment by zaazbb** [comment]
and, the release tar (on the release web page, not on pypi), should include icons, and portable.
so, i can unzip it , run it directly.

**Comment by mitya57** [comment]
You should be already able to run ReText directly from the tarball. It will find the icons, translations, etc. in their current location (so no need to move anything).

The official tarball is on PyPI. What is on GitHub is generated automatically from Git repository and does not contain generated files.

**Comment by zaazbb** [comment]
no, if i do not move data files to share dir, and do not add the sourc- root-dir/share/... to datadirs, it can not disp tool button icons.

**Comment by mitya57** [comment]
Does this still happen with the latest official tarball ([ReText-7.0.3.tar.gz](https://files.pythonhosted.org/packages/01/70/aa870c2222be0ffe6efc700d9eb4f9125164b04e017a00f9302774ad62b2/ReText-7.0.3.tar.gz))?
